### PR TITLE
#174 - Add IT for installation deprecated package from dependency

### PR DESCRIPTION
**Pull request by genryxy**
Part of #174
Add IT for installation deprecated package from dependency

### CODE REVIEW

File: src/test/java/com/artipie/npm/NpmDeprecateIT.java [hunk 1]
```diff
@@ -166,9 +166,44 @@ void installsWithDeprecationWarning() throws Exception {
...
+        final String proj = "@hello/simple-npm-project";
+        final String withdep = "project-with-simple-dependency";
+        new TestResource("simple-npm-project")
+            .addFilesTo(this.storage, new Key.From(String.format("tmp/%s", proj)));
+        new TestResource(withdep)
+            .addFilesTo(this.storage, new Key.From(String.format("tmp/%s", withdep)));
+        this.exec("npm", "publish", String.format("tmp/%s", proj), "--registry", this.url);
+        this.exec("npm", "publish", String.format("tmp/%s", withdep), "--registry", this.url);
+        final String msg = "Danger! Do not use!";
+        this.exec("npm", "deprecate", proj, msg, "--registry", this.url);
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/npm/NpmDeprecateIT.java:180]: @genryxy what if msg is empty? I'd suggest to check this case too

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@genryxy one suggestion, please, check

**Comment by genryxy** [comment]
@olenagerasimova thanks, I fixed deleting `deprecated` field from `meta.json` for message `""` and added test.

**Review by olenagerasimova — APPROVED** [review]
@genryxy thanks, looks good to merge

**Comment by 0crat** [comment]
Job `gh:artipie/npm-adapter#178` is not assigned, can't get performer

**Pull request merged**